stdlib/task/src/main/ballerina/src/task/listener.bal

```diff
@@ -111,14 +111,14 @@ public type Listener object {
 
     # Pauses the `task:Listener` and the attached services.
     #
-    # + return - Returns `task:ListenerError` if an error is occurred while resuming or nil Otherwise
+    # + return - `task:ListenerError` if an error is occurred while pausing or nil otherwise
     public function pause() returns ListenerError? {
         return pauseExternal(self);
     }
 
     # Resumes a paused `task:Listener`. Calling this on an already running `task:Listener` will not cause any error.
     #
-    # + return - Returns `task:ListenerError` when an error occurred while pausing or nil Otherwise
+    # + return - `task:ListenerError` when an error occurred while resuming or nil otherwise
```

Comment:
```suggestion
    # + return - `task:ListenerError` when an error occurred while resuming or nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.